NEWPART.md

```diff
@@ -0,0 +1,203 @@
+
+### Step 5 ###
+
+Now we'll handle the extra parts -- those not fully bonded out.   These are usually the parts you actually have on the boards you buy.
+
+* Add a line in the appropriate 'harness' section for any alternative packages (typically with fewer pins, in this case, 324 versus 676).   If any XRAY\_PIN\_XX values you listed in the settings file are not bonded out on the new part, you must specify alternatives.   In this example, we need to specify a new clock pin, `XRAY_PIN_00=N15`.
+
+```
+db-extras-artix7-harness:
+        +XRAY_PIN_00=J13 XRAY_PIN_01=J14 XRAY_PIN_02=K15 XRAY_PIN_03=K16 \
+                XRAY_PART=xc7a35tftg256-1 XRAY_EQUIV_PART=xc7a50tfgg484-1 $(MAKE) -C fuzzers roi_only
++       +source settings/artix7_100t.sh && \
++               XRAY_PIN_00=N15 XRAY_PART=xc7a100tcsg324-1 XRAY_EQUIV_PART=xc7a100tfgg676-1 \
++               $(MAKE) -C fuzzers roi_only
+        +source settings/artix7_200t.sh && \
+                XRAY_PIN_00=V10 XRAY_PIN_01=W10 XRAY_PIN_02=Y11 XRAY_PIN_03=Y12 \
+                XRAY_PART=xc7a200tsbg484-1 XRAY_EQUIV_PART=xc7a200tffg1156-1 \
+                $(MAKE) -C fuzzers roi_only
+
+```
+
+Make the appropriate harness target:
+
+```
+source settings/artix7.sh
+make -j32 db-extras-artix7-harness
+```
+
+This target makes stuff for the extra parts in all the artix7 device families, not just your new device.
+
+
+### Step 6 ###
+
+Do a spot check.
+
+* Check that there are new "part" directories:
+
+```
+$ ll database/artix7/xc7a100*
+database/artix7/xc7a100tcsg324-1:
+total 19884
+drwxrwxr-x  2 tcal tcal     4096 Apr 29 08:01 ./
+drwxrwxr-x 13 tcal tcal    32768 Apr 29 08:00 ../
+-rw-rw-r--  1 tcal tcal    10364 Apr 29 08:00 package_pins.csv
+-rw-rw-r--  1 tcal tcal    32142 Apr 29 08:01 part.json
+-rw-rw-r--  1 tcal tcal    22440 Apr 29 08:01 part.yaml
+-rw-rw-r--  1 tcal tcal  8601612 Apr 29 08:01 tileconn.json
+-rw-rw-r--  1 tcal tcal 11648042 Apr 29 08:01 tilegrid.json
+
+database/artix7/xc7a100tfgg676-1:
+total 19892
+drwxrwxr-x  2 tcal tcal     4096 Apr 29 02:03 ./
+drwxrwxr-x 13 tcal tcal    32768 Apr 29 08:00 ../
+-rw-rw-r--  1 tcal tcal    16645 Apr 28 22:16 package_pins.csv
+-rw-rw-r--  1 tcal tcal    32165 Apr 28 22:17 part.json
+-rw-rw-r--  1 tcal tcal    22440 Apr 28 22:17 part.yaml
+-rw-rw-r--  1 tcal tcal  8601612 Apr 29 02:03 tileconn.json
+-rw-rw-r--  1 tcal tcal 11648042 Apr 28 22:37 tilegrid.json
+```
+
+In this case, the tile grid is the same size since it's the same chip, but the size of the  package pins files differs since differing numbers of pins are bonded.
+
+
+### Step 7 ###
+
+Assuming everything looks good, commit to your fork.   You should have a new file under settings/, a new ignored wires file, and a modified Makefile.
+
+```
+git add Makefile settings/artix7_100t.sh
+git status
+git commit --signoff 
+```
+
+### Step 8 ###
+
+Push to GitHub:
+```
+git push origin master  // or whatever branch you're using
```

Comment:
I generally recommend that users create a new branch when submitting PRs with a useful branch name.

Reply:
I'm updating the instructions to create a branch with a useful name.